---
title: "1000 Genomeæ•°æ®åº“å­¦ä¹ ç¬”è®°"
date: 2021-09-13T11:20:55+09:00
draft: false
cover: "/img/1KG.jpg"
categories:
  - å­¦ä¹ ç¬”è®°ğŸ’»
tags:
  - Bioinfo
  - æ•°æ®åº“
---

---

<!--more-->

# **1000 GENOME**

[1000 Genomes | A Deep Catalog of Human Genetic Variation](https://www.internationalgenome.org/)

## ä¸€ã€ç®€ä»‹

åƒäººåŸºå› ç»„è®¡åˆ’ï¼ˆ1000 Genomesï¼Œ[http://www.internationalgenome.org/](http://www.internationalgenome.org/)Â ï¼Œç®€ç§°1KGï¼‰äº2008-2015å¹´å¼€å¯çš„å¯¹äººçš„åŸºå› ç»„è¿›è¡Œæµ‹åºä¸€ä¸ªé¡¹ç›®ï¼Œç›®çš„æ˜¯å»ºç«‹äººç±»çªå˜å’Œåˆ†å‹çš„å…±åŒæ•°æ®åº“ã€‚è™½ç„¶è¿™ä¸ªé¡¹ç›®å·²ç»ç»“æŸäº†ï¼ŒEMBL-EBIçš„æ•°æ®ä¸­å¿ƒä»ç„¶è·å¾—äº†Wellcome Trustçš„åŸºé‡‘èµ„åŠ©æ¥ç»´æŠ¤å’Œæ‰©å……è¿™ä¸ªæ•°æ®åº“ã€‚

 IGSRï¼ˆInternational Genome Sample Resourceï¼‰æƒ³å®ç°çš„ç›®æ ‡ï¼š

- ä¿è¯å…¬ä¼—èƒ½è®¿é—®å’Œä½¿ç”¨1000 Genomesçš„æ•°æ®
- è¡¥å……å·²å‘è¡¨çš„åŸºå› ç»„å…¶ä»–ä¿¡æ¯
- å‘1000 GenomesæŒç»­å¢åŠ æ–°çš„åŸºå› ç»„æ•°æ®

é¡¹ç›®å®Œæˆçš„ä¸‰ä¸ªé˜¶æ®µï¼š

|é˜¶æ®µ|ç›®æ ‡|æµ‹åºæ·±åº¦|æ ·æœ¬|å®Œæˆæ—¶é—´|
|:---:|:---:|:---:|:---:|:---:|
|1-low coverage|Assess strategy of sharing data across samples|2-4X|180ä¸ªæ ·æœ¬å…¨åŸºå› ç»„æµ‹åº|2008å¹´9æœˆ|
|2-trios|Assess coverage and platforms and centres|20-60X|2ä¸ªæ¯äº²-çˆ¶äº²-å­©å­çš„å®¶ç³»æµ‹åº|2008å¹´10æœˆ|
|3-gene regions|Assess methods for gene-region-capture|50X|900ä¸ªæ ·æœ¬1000ä¸ªåŸºå› |2009å¹´6æœˆ|

è¿™ä¸ªé¡¹ç›®åŒ…æ‹¬5ä¸ªå¤§çš„äººç§ï¼Œ25ä¸ªäºšäººç§ã€‚ç›®å‰ï¼Œæ–°ç‰ˆå…±æœ‰NAç¼–å·å¼€å¤´çš„1182ä¸ªäººï¼ŒHGå¼€å¤´çš„1768ä¸ªäººã€‚1KGé¡¹ç›®ä¸HapMapé¡¹ç›®å…±äº«ä¸€éƒ¨åˆ†æ ·æœ¬ï¼Œå…¶ä¸­NAç¼–å·å¼€å¤´çš„æ ·æœ¬å¾ˆæœ‰å¯èƒ½å°±æ˜¯æ¥è‡ªHapMapé¡¹ç›®çš„æ ·æœ¬ã€‚

1KGè®¡åˆ’çš„æœ€åé˜¶æ®µæ˜¯ç¬¬ä¸‰é˜¶æ®µï¼Œä»£è¡¨GRCh37ä¸Šçš„2504ä¸ªæ ·æœ¬ã€‚éšååœ¨GRCh38ä¸Šé‡æ–°åˆ†æäº†ç¬¬ä¸‰é˜¶æ®µçš„æ•°æ®ã€‚åœ¨è¿™é¡¹å·¥ä½œä¹‹åï¼Œæ ·æœ¬å·²è¢«é‡æ–°æ’åˆ—ä¸ºé«˜è¦†ç›–ç‡ï¼Œå¹¶ä¸”å¯¹å…¶ä»–ç›¸å…³æ ·æœ¬è¿›è¡Œäº†æµ‹åºï¼Œæ ·æœ¬æ€»æ•°è¾¾åˆ°3202ä¸ªã€‚

ç¬¬ä¸‰é˜¶æ®µçš„æ•°æ®é›†è¢«è®¤ä¸ºæ˜¯1KGè®¡åˆ’çš„æœ€ç»ˆæ•°æ®å‘å¸ƒç‰ˆæœ¬ã€‚ç¬¬ä¸€é˜¶æ®µæ˜¯ä¸€ä¸ªè¯•ç‚¹ï¼Œåœ¨æœ‰é™æ•°é‡çš„å—è¯•è€…ä¸Šæµ‹è¯•äº†ä¸åŒçš„æµ‹åºå’Œæ¯”å¯¹æ–¹æ³•ï¼Œè¿›è€Œä½¿å¾—ç¬¬ä¸‰é˜¶æ®µçš„æ–¹æ³•æ›´åŠ æ ‡å‡†åŒ–ã€‚å¦å¤–ï¼Œç¬¬ä¸‰é˜¶æ®µåŒ…å«æ¥è‡ªæ›´å¤šç§ç¾¤çš„æ•°æ®ã€‚å…¶ä¸­ç¬¬ä¸€é˜¶æ®µæœ‰59åå—è¯•è€…æ²¡æœ‰åŒ…å«åœ¨ç¬¬ä¸‰é˜¶æ®µï¼Œå› ä¸ºä»–ä»¬çš„ä½è¦†ç›–ç‡æµ‹åºå’Œ/æˆ–å¤–æ˜¾å­æ•°æ®ä¸ç¬¦åˆç¬¬ä¸‰é˜¶æ®µçš„æ ‡å‡†ã€‚

ç¬¬ä¸‰é˜¶æ®µçš„releaseåŒ…å«æ¥è‡ª26ä¸ªç§ç¾¤çš„2504ä¸ªæ ·æœ¬ï¼Œå¯ä»¥æŒ‰ç…§å¤§é™†åˆ†ä¸º5ä¸ªsuper populationï¼š

- ä¸œäºš EAS
- å—äºš SAS
- éæ´² AFR
- æ¬§æ´² EUR
- ç¾å›½ AMR

æ¯ä¸ªæ ·æœ¬çš„super populationä¿¡æ¯å¯ä»¥åœ¨integrated_call_samples_v3.20130502.ALL.panelä¸­æ‰¾åˆ°ã€‚

å®ƒçš„å®˜æ–¹ç½‘ç«™æœ‰ä¸€ä¸ªpptè®²å¾—å¾ˆæ¸…æ¥šå¦‚ä½•é€šè¿‡å®˜ç½‘åšçš„data portalæ¥ä¸‹è½½æ•°æ®ï¼š

[How_to_Access_the_Data.pdf](https://www.genome.gov/Pages/Research/DER/ICHG-1000GenomesTutorial/How_to_Access_the_Data.pdf)

å› ä¸ºæ•°æ®åº“æ˜¯å…è´¹å…¬å¼€æä¾›çš„ï¼Œæ‰€ä»¥æ²¡æœ‰æ”¶é›†è¡¨å‹ä¿¡æ¯ï¼Œ[ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/working/20130606_sample_info/20130606_g1k.ped](ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/working/20130606_sample_info/20130606_g1k.ped) æä¾›æ ·æœ¬çš„IDï¼Œç§ç¾¤ï¼Œæ€§åˆ«ï¼Œå®¶æ—å…³ç³»çš„ä¿¡æ¯ã€‚

æ ·æœ¬ä¸ªä½“çš„ç§ç¾¤ï¼ˆå›½å®¶åŠåœ°åŒºï¼‰éƒ½æœ‰ç›¸å¯¹åº”çš„ä»£ç ï¼Œå¯¹åº”è¯´æ˜å¦‚ä¸‹ï¼š

### Populations

ThisÂ fileÂ describesÂ theÂ populationÂ codesÂ whereÂ assignedÂ toÂ samplesÂ collectedÂ forÂ theÂ 1000Â GenomesÂ project.Â TheseÂ codesÂ areÂ usedÂ toÂ organiseÂ theÂ filesÂ inÂ theÂ data_collections'Â projectÂ dataÂ directoriesÂ andÂ canÂ alsoÂ beÂ foundÂ inÂ columnÂ 11Â ofÂ manyÂ sequenceÂ indexÂ files.

ThereÂ areÂ alsoÂ twoÂ tsvÂ files,Â whichÂ containÂ theÂ populationÂ codesÂ andÂ descriptionsÂ forÂ bothÂ theÂ subÂ andÂ superÂ populationsÂ thatÂ wereÂ usedÂ inÂ phaseÂ 3Â ofÂ theÂ 1000Â GenomesÂ Project:

ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/20131219.populations.tsv

ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/20131219.superpopulations.tsv

### PopulationsÂ codes

```
         CHB	Han Chinese             Han Chinese in Beijing, China
         JPT	Japanese                Japanese in Tokyo, Japan
         CHS	Southern Han Chinese    Han Chinese South
         CDX	Dai Chinese             Chinese Dai in Xishuangbanna, China
         KHV	Kinh Vietnamese         Kinh in Ho Chi Minh City, Vietnam
         CHD	Denver Chinese          Chinese in Denver, Colorado (pilot 3 only)
	
         CEU	CEPH                    Utah residents (CEPH) with Northern and Western European ancestry 
         TSI	Tuscan                  Toscani in Italia 
         GBR	British                 British in England and Scotland 
         FIN	Finnish                 Finnish in Finland 
         IBS	Spanish                 Iberian populations in Spain 
	
         YRI	Yoruba                  Yoruba in Ibadan, Nigeria
         LWK	Luhya                   Luhya in Webuye, Kenya
         GWD	Gambian                 Gambian in Western Division, The Gambia 
         MSL	Mende                   Mende in Sierra Leone
         ESN	Esan                    Esan in Nigeria
	
         ASW	African-American SW     African Ancestry in Southwest US  
         ACB	African-Caribbean       African Caribbean in Barbados
         MXL	Mexican-American        Mexican Ancestry in Los Angeles, California
         PUR	Puerto Rican            Puerto Rican in Puerto Rico
         CLM	Colombian               Colombian in Medellin, Colombia
         PEL	Peruvian                Peruvian in Lima, Peru

         GIH	Gujarati                Gujarati Indian in Houston, TX
         PJL	Punjabi                 Punjabi in Lahore, Pakistan
         BEB	Bengali                 Bengali in Bangladesh
         STU	Sri Lankan              Sri Lankan Tamil in the UK
         ITU	Indian                  Indian Telugu in the UK
```

## äºŒã€æ•°æ®æŸ¥è¯¢ä¸ä¸‹è½½

[Data | 1000 Genomes](https://www.internationalgenome.org/data)

### æŸ¥è¯¢æ•°æ®

- é€šè¿‡æµè§ˆå™¨ç›´æ¥æŸ¥è¯¢ï¼š

    åƒäººåŸºå› ç»„è®¡åˆ’ -- åŸºå› ç»„æµè§ˆå™¨ï¼šÂ [1000 Genomes Browser](http://www.ncbi.nlm.nih.gov/variation/tools/1000genomes/)

- æŸ¥è¯¢æŸä¸ªSNPçš„ä¿¡æ¯ï¼Œåªè¦å°†SNPçš„rsç¼–å·æŒ‰ä¸‹é¢çš„æ ¼å¼è¾“å…¥åœ°å€æ å°±å¯ä»¥ï¼š

```
http://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs35761398
http://www.ncbi.nlm.nih.gov/SNP/snp_ref.cgi?rs=2501432
```

å¯ä»¥æŒ‰ç…§äººç§æ¥æŸ¥çœ‹è¿™äº›æ•°æ®ï¼šftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000_genomes_project/data/

æ¯ä¸ªäººçš„ç›®å½•ä¸‹é¢éƒ½æœ‰ å››ä¸ªæ•°æ®æ–‡ä»¶å¤¹

```
Oct 01 2014 00:00    Directory alignment
Oct 01 2014 00:00    Directory exome_alignment
Oct 01 2014 00:00    Directory high_coverage_alignment
Oct 01 2014 00:00    Directory sequence_read
```

å¯ä»¥ç›´æ¥çœ‹æœ€æ–°ç‰ˆçš„vcfæ–‡ä»¶ï¼Œè®°å½•äº†è¿™ä¸¤åƒå¤šäººçš„æ‰€æœ‰å˜å¼‚ä½ç‚¹ä¿¡æ¯ã€‚å¯ä»¥ç›´æ¥çœ‹åˆ°æ‰€æœ‰çš„ä½ç‚¹ï¼Œå…·ä½“åˆ°æ¯ä¸ªäººåœ¨è¯¥ä½ç‚¹æ˜¯å¦å˜å¼‚ï¼šftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/

å®ƒçš„åŸºå› å‹ä¿¡æ¯æ˜¯é€šè¿‡MVNcall+SHAPEITè¿™ä¸ªç¨‹åºcallå‡ºæ¥çš„ï¼Œå…·ä½“åŸç†è§ï¼š[http://www.ncbi.nlm.nih.gov/pubmed/23093610](http://www.ncbi.nlm.nih.gov/pubmed/23093610)

è€Œä¸”ç½‘ç«™è¿˜æä¾›ä¸€äº›è¯´æ˜ï¼šftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000_genomes_project/working/

### ä¸‹è½½æ•°æ®

ä¸‹è½½æ•°æ®å¯ä»¥é€šè¿‡FTPç«™ç‚¹è¿›è¡Œä¸‹è½½ã€‚

- 1KGçš„FTPç«™ç‚¹ï¼š

    [http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/)

- NCBI FTP Site :

    [ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp](ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp)

- Amazon S3 :

    s3://1000genomes

æˆ‘ä»…ä½¿ç”¨è¿‡å‰ä¸¤ä¸ªç«™ç‚¹ï¼Œå†…å®¹åº”è¯¥éƒ½æ˜¯ä¸€æ ·çš„ã€‚

â—æ³¨æ„ï¼šåœ¨Linuxç³»ç»Ÿæ“ä½œæ—¶ï¼Œå¯ä»¥ä½¿ç”¨wgetå‘½ä»¤è¿›è¡Œæ–‡ä»¶ä¸‹è½½ï¼Œæ ¼å¼æ˜¯`wget [Link]`ï¼Œè¿™é‡Œè¦ä½¿ç”¨ftpåœ°å€è€Œä¸æ˜¯httpåœ°å€ã€‚


## ä¸‰ã€è¡¨è¾¾ç›¸å…³æ•°æ®

### Geuvadisçš„RNAæµ‹åºé¡¹ç›®

[http://www.geuvadis.org](http://www.geuvadis.org/)

å…³äºæµ‹åºç»†èŠ‚çš„è®ºæ–‡ï¼š[Lappalainen et al. Nature 2013](http://dx.doi.org/10.1038/nature12531)
åŒ…å«465ä¸ªä¸ªä½“ï¼ˆåŒ…æ‹¬ç§ç¾¤ï¼šCEU, TSI, GBR, FIN, YRIï¼‰çš„RNAseqæ•°æ®ï¼ˆmRNAå’ŒmiRNAï¼‰

ï¼ˆä½†æ˜¯è¯´æ˜é‡Œæ˜¯462 unrelated human lymphoblastoid cell line samplesï¼Œå’Œå®˜ç½‘æ ‡é¢˜é‡Œçš„465æ ·æœ¬æœ‰å·®åˆ«ï¼Œè¿˜æ²¡å¼„æ¸…æ¥šä¸ºä»€ä¹ˆï¼‰

```
CEUï¼šå…·æœ‰åŒ—æ¬§å’Œè¥¿æ¬§è¡€ç»Ÿçš„çŠ¹ä»–å·å±…æ°‘ (CEPH)ã€‚
TSIï¼šæ„å¤§åˆ©çš„Toscaniäººã€‚
GBRï¼šè‹±æ ¼å…°å’Œè‹æ ¼å…°çš„è‹±å›½äººã€‚
FINï¼šèŠ¬å…°çš„èŠ¬å…°äººã€‚
YRIï¼šå°¼æ—¥åˆ©äºšçš„Ibadançš„Yorubaäººã€‚
```

è¿™ä¸ªé¡¹ç›®åŒ…å«ä¸‰ä¸ªé˜¶æ®µçš„ä¸åŒæµ‹åºæ•°æ®ï¼Œé˜¶æ®µä¸‰åº”è¯¥æ˜¯æœ€å…¨çš„ï¼š

- mRNAæµ‹åºæ•°æ®ï¼š[http://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-1/samples.html](http://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-1/samples.html) 

- miRNAæµ‹åºæ•°æ®ï¼š[http://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-2/samples.html](http://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-2/samples.html) 

- å®Œæ•´çš„åŒ…å«mRNAå’ŒmiRNAæµ‹åºæ•°æ®ï¼š[https://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-3/](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-3/)  

#### Bamæ ¼å¼æ•°æ®

é¡¹ç›®æä¾›å·²ç»å¤„ç†è¿‡çš„æ•°æ®ï¼ˆbamæ ¼å¼ï¼‰ä¸‹è½½ï¼š

Processed Data: [https://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-3/files/processed/](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-3/files/processed/)

ï¼ˆhuman hg19 reference genome (autosomes+X+Y+M) with the GEM mapper v 1.349 mapping entire readsï¼‰

#### eQTLç»“æœæ•°æ®

è¡¨è¾¾æ•°æ®åˆ†æç»“æœæ•°æ®â€”â€”eQTLï¼ˆè¡¨è¾¾æ•°é‡æ€§çŠ¶åŸºå› åº§ï¼‰ç»“æœä¸‹è½½ï¼š

[https://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-3/files/analysis_results/](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEUV-3/files/analysis_results/)

ä½†æ˜¯è¿™é‡Œçš„æ•°æ®æ˜¯æ‹†åˆ†å¼€çš„ï¼Œåˆ†ä¸ºEURå’ŒYRIï¼Œæ¬§æ´²å¤§é™†4ä¸ªç§ç¾¤å’Œéæ´²å¤§é™†çš„å°¼æ—¥åˆ©äºšç§ç¾¤ã€‚

æ•°é‡æ€§çŠ¶åŒ…æ‹¬ï¼šå¤–æ˜¾å­ï¼ŒåŸºå› ï¼Œè½¬å½•ç‡ï¼Œè½¬å½•çš„repetitive elementï¼ŒmiRNAã€‚æ–‡ä»¶ä¸­åŒ…å«False discovery rateä½äº5%çš„æ‰€æœ‰å…³è”associationã€‚å¦‚æœæœ‰å¤šä¸ªç›¸åŒpå€¼çš„æœ€ä¼˜å…³è”å˜ä½“ï¼Œéšæœºé€‰æ‹©å…¶ä¸­ä¹‹ä¸€ã€‚

ä¸‹é¢æ˜¯å®˜æ–¹å¯¹eQTLæ–‡ä»¶çš„ä¸€äº›è¯´æ˜ï¼š

eQTL file set:

- Sample set & sample size : EUR373, YRI89
- Quantitative trait: exon, gene, transcript ratio (trratio), transcribed repetitive element (repeat), miRNA (mi)
- Set of associations included in the file: All the associations below false discovery rate 5%, or best association per each gene (for exon, transcript ratio) or unit (gene, repeat, miRNA). If there are several best associating variants with the same p-value, one of them has been chosen randomly.

eQTL file format:

- The file contains variantÃ¢â‚¬â€œQT association information, with the following columns:
1	SNP_ID : Variant identifier according to dbSNP137; position-based identifier for variants that are not in dbSNP (see Supplementary material pp 45)
2	ID : Null (-)
3	GENE_ID : Gene identifier according to Gencode v12, miRBase v18, repeats based on their start site
4	PROBE_ID : Quantitative trait identifier; the same as GENE_ID expect for:
Exons: GENEID_ExonStartPosition_ExonEndPosition
Transcript ratios: Transcript identifier according to Gencode v12
5	CHR_SNP : Chromosome of the variant
6	CHR_GENE : Chromosome of the quantitative trait
7	SNPpos : Position of the variant
8	TSSpos : Transcription start site of the gene/QT
9	Distance : | SNPpos Ã¢â‚¬â€œ TSSpos |
10	rvalue	: Spearman rank correlation rho (calculated from linear regression slope). The sign denotes the direction of the nonreference allele (i.e. rvalue<0 means that nonreference allele has lower expression)
11	pvalue : Association p-value
12	log10pvalue : -log10 of pvalue

### å…¶ä»–è¡¨è¾¾æ•°æ®

1. 60ä¸ªCEUä¸ªä¸ªä½“RNAseq

    [http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-197](http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-197)

2. 800 HapMapä¸ªä½“çš„è¡¨è¾¾èŠ¯ç‰‡

    [http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-198](http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-198)

    [http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-264](http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-264)

3. 69ä¸ªYRIä¸ªä½“çš„RNAseqæ•°æ®

    [http://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-19480](http://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-19480)


---